mod/deckorate_attribution/set/type/reference/tabs_placeholder.haml

```diff
@@ -0,0 +1,10 @@
+%ul.tabs
+  %li.tab.tab-placeholder Rich Text
+  %li.tab.tab-placeholder Plain Text
+  %li.tab.tab-placeholder HTML
+
+.d-flex.mb-3.align-items-stretch.pt-3
+  .p.clipboard.ps-3.p-2.rounded-start.flex-fill
+    Get the Wikirate.org Attribution Notice
+  - save_icon = icon_tag :save
+  = standard_save_button({ "text": "Save #{save_icon}".html_safe })
```

Comment:
We don't want to just hard-code html that we're already generating elsewhere. Please see how we generate tabs